### DISCUSSION TITLE
Maintenance reminders

### DISCUSSION THREAD

**Discussion by vadikcoma** [teslamate-org/teslamate#2910] [Ideas]
Is it possible to have email with maintenance reminders based on mileage/time? Tire rotation, air filters...

**Comment by cwanja** [comment]
Had the same idea in #245 😊

**Comment by sultantariq** [comment]
> Is it possible to have email with maintenance reminders based on mileage/time? Tire rotation, air filters...

You can use nodered to get an email/telegram as a reminder.
